docs/hub/datasets-fiftyone.md

```diff
@@ -0,0 +1,144 @@
+# FiftyOne
+
+FiftyOne is the leading open-source toolkit for curating, visualizing, and
+managing unstructured visual data. The library streamlines data-centric
+workflows, from finding low-confidence predictions to identifying poor-quality
+samples and uncovering hidden patterns in your data. The library supports all
+sorts of visual data, from images and videos to PDFs, point clouds, and meshes.
+
+Whereas tabular data formats like a pandas DataFrame or a Parquet file consist
+of rows and columns, FiftyOne datasets are considerably more flexible,
+accomodating object detections, keypoints, polylines, etc. and custom schemas.
+
+FiftyOne is integrated with the Hugging Face Hub, so you can load and share
+FiftyOne datasets directly from the Hub.
+
+## Prerequisites
+
+First [login with your Hugging Face account](../huggingface_hub/quick-start#login):
+
+```bash
+huggingface-cli login
+```
+
+Make sure you have `fiftyone>=0.24.0` installed:
+
+```bash
+pip install -U fiftyone
+```
+
+## Loading Visual Datasets from the Hub
+
+With `load_from_hub()` from FiftyOne's Hugging Face utils, you can load:
+
+- Any FiftyOne dataset uploaded to the hub
+- Most image-based datasets stored in Parquet files (which is the standard for datasets uploaded to the hub via the `datasets` library)
+
+### Loading FiftyOne datasets from the Hub
+
+Any dataset pushed to the hub in one of FiftyOne’s [supported common formats](https://docs.voxel51.com/user_guide/dataset_creation/datasets.html#supported-import-formats)
+should have all of the necessary configuration info in its dataset repo on the
+hub, so you can load the dataset by specifying its `repo_id`. As an example, to
+load the [VisDrone detection dataset](https://huggingface.co/datasets/Voxel51/VisDrone2019-DET),
+all you need is:
+
+```python
+import fiftyone as fo
+from fiftyone.utils import load_from_hub
+
+## load from the hub
+dataset = load_from_hub("Voxel51/VisDrone2019-DET")
+
+## visualize in app
+session = fo.launch_app(dataset)
+```
```

Comment:
Is there a link to a google colab or a HF space for people to play with the interface ?

Reply:
Here's a Colab notebook showcasing most of the functionality: https://colab.research.google.com/drive/1l0kzfbJ2wtUw1EGS1tq1PJYoWenMlihp?usp=sharing

Should I link to this in some way from the docs page?